sys-block/gpart/gpart-0.3.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A small tool which tries to guess what partitions are on a hard disk"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
+HOMEPAGE="https://github.com/baruch/gpart"
+COPYRIGHT="1999-2017 Michail Brzitwa"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/baruch/gpart/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="ec56d12ec9ffdb9877c12692ea6e51620b1ae44473d3d253b27fc31ed9ebb4dd"
+PATCHES="
+	gpart-$portVersion.patchset.1
+	gpart-$portVersion.patchset.2
+	"
+
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	gpart = $portVersion
+	cmd:gpart = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autom4te
+	cmd:automake
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc
+	cmd:grep
+	cmd:ld
+	cmd:make
+	cmd:makeinfo
+	cmd:pkg_config
+	cmd:sed
+	"
+
+BUILD()
+{
+	NOCONFIGURE=1 ./autogen.sh
```

Comment:
I do wish `NOCONFIGURE` was a thing [there](https://github.com/baruch/gpart/blob/23b651d1e9c6ae1454e5d3fe315d7d4e145661c0/autogen.sh#L7). I think your best bet right now is `autoreconf -vfi`